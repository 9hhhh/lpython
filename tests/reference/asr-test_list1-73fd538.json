{
    "basename": "asr-test_list1-73fd538",
    "cmd": "lpython --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/errors/test_list1.py",
    "infile_hash": "86f36c119afcaef5c6f790c88c7c517fa6f766bfcc95d121bc16becd",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-test_list1-73fd538.stderr",
    "stderr_hash": "4bda0c0451782d4d487da271e484718ac1efc9fbe001e4d8e122bfc0",
    "returncode": 2
}