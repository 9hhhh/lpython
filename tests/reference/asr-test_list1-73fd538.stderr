semantic error: Type mismatch while removing from a list, found (character and integer).
 --> tests/errors/test_list1.py:6:5
  |
6 |     a.remove('error')
  |     ^^^^^^^^^^^^^^^^^ 
